gnovm/pkg/gnolang/machine.go

```diff
@@ -252,6 +252,12 @@ func (m *Machine) runMemPackage(mpkg *std.MemPackage, save, overrides bool) (*Pa
 	mpkg.Sort()
 	// parse files.
 	files := ParseMemPackage(mpkg)
+	mod, err := ParseCheckGnoMod(mpkg)
```

Comment:
It has already been verified at the addpkg time. Let's use a lighter version that won't "check," just parse.